### ISSUE TITLE
System.ValueTuple package is missing net472 target directory

### ISSUE THREAD

**Issue by tmat** [dotnet/runtime#29679]
When targeting .NET Framework 4.7.2 RAR claims the project should copy System.ValueTuple.dll from the package (from net47 directory) to its output dir. But in reality it shouldn’t copy anything since it’s already in the FX.

System.ValueTuple package has netstandard2.0 target dir containing `_._` sentinel, but does not have one for `net472`. Then targeting .NET Framework RAR falls back to `net47`, which in the package has `System.ValueTuple.dll`. 

@ericstj points out that conflict resolution does not handle this case - it’s losing due to assembly versions:
```
Encountered conflict between 'Reference:C:\Users\ericstj\.nuget\packages\system.valuetuple\4.5.0\ref\net47\System.ValueTuple.dll' and 'Platform:System.ValueTuple.dll'.  NETSDK1033: Choosing 'Reference:C:\Users\ericstj\.nuget\packages\system.valuetuple\4.5.0\ref\net47\System.ValueTuple.dll' because AssemblyVersion '4.0.3.0' is greater than '4.0.2.0'.
Encountered conflict between 'Platform:System.ValueTuple.dll' and 'CopyLocal:C:\Users\ericstj\.nuget\packages\system.valuetuple\4.5.0\lib\net47\System.ValueTuple.dll'.  NETSDK1033: Choosing 'CopyLocal:C:\Users\ericstj\.nuget\packages\system.valuetuple\4.5.0\lib\net47\System.ValueTuple.dll' because AssemblyVersion '4.0.3.0' is greater than '4.0.2.0'.
```

**Comment by joperezr** [comment]
@ericstj did this ever got fixed in 2.1?

**Comment by jeffhandley** [comment]
I'm closing this issue out as being stale.
